std/merge.js

```diff
@@ -81,6 +81,243 @@ function objectMerge(obj, mergeObj) {
   return r;
 }
 
+function mergeFunc(rule, key, defaultFunc) {
+  const f = rule && rule[key];
+  if (f === undefined) {
+    return defaultFunc;
+  }
+
+  if (typeof f !== 'function') {
```

Comment:
Suggestion:  take an object here to be synonymous with `deep(object)`

Reply:
Indeed! that's a nice suggestion you had earlier and that's point 1. in the TODO list :)